src/Base/PitchClass.hs

```diff
@@ -17,15 +21,23 @@ pitchClass i = PitchClass $ i `mod` 12
 shiftPitchClassBy :: Int -> PitchClass -> PitchClass
 shiftPitchClassBy by (PitchClass i) = pitchClass $ by + i
 
-noteToPitchClass :: Note -> PitchClass
-noteToPitchClass C = pitchClass 0
-noteToPitchClass D = pitchClass 2
-noteToPitchClass E = pitchClass 4
-noteToPitchClass F = pitchClass 5
-noteToPitchClass G = pitchClass 7
-noteToPitchClass A = pitchClass 9
-noteToPitchClass B = pitchClass 11
-
-rootToPitchClass :: Root -> PitchClass
-rootToPitchClass r =
-  shiftPitchClassBy (impliedShift $ getAcc r) (noteToPitchClass $ getRoot r)
+infixl 6 @-@
+(@-@) :: PitchClass -> PitchClass -> Int
+(PitchClass p1) @-@ (PitchClass p2) = (p1 - p2) `mod` 12
```

Comment:
Since I don't remember how Haskell's `mod` function behaves, just make sure this acutally does what you want (i.e. always returns a positive value.)

Reply:
Just did some real quick tests, and it does always return a positive value. I think `rem` is the one that could return a negative value.

Reply:
Nice!